### DISCUSSION TITLE
Web version?

### DISCUSSION THREAD

**Discussion by PatrikGolubar** [flutter-stripe/flutter_stripe#1063] [Q&A]
Hi,
When it will be enabled to use "Future Web Payments" with this library? In example app it shows that is unsupported for web yet..

**Comment by sleewok** [comment]
Great question. It was "supposedly" being worked on a couple weeks ago but there has been silence since then. I have offered to help, but have not had any response. It is pretty disappointing really.

**Reply by jonasbark** [reply]
We're looking forward to your contributions! 
We (the three main maintainers) do this in our free time and that time is limited. I'm sure you understand that.

**Reply by PatrikGolubar** [reply]
Thank you for your answer. Will you prepare update in february? I need this update to complete app for my customer 
Do you have any faster way of communications?

**Reply by sleewok** [reply]
> We're looking forward to your contributions! We (the three main maintainers) do this in our free time and that time is limited. I'm sure you understand that.

Thanks for the quick response! I would really like to help with the confirmSetupIntent (issue #960)  and help get it working for future sessions. I have been using Flutter for a couple years, but I have yet to develop any packages and it is a bit overwhelming trying to track down where to begin trying to work on this. I would be very thankful for any guidance on getting started. In particular I was initially trying to track down where it was actually sending the parameters to Stripe since the parameter is incorrect (paymentMethod vs payment_method).

**Reply by PatrikGolubar** [reply]
Hi, is there any new update for this?

**Reply by tedfire** [reply]
Hey, could you at least give an example of how to use flutter_stripe_web, its so tantilizing there ! And I really, really need to use it. I know you guys are stacked. Any docs you can scrape together appreciated.

**Reply by tedfire** [reply]
The google dev team should really be supporting this. They have a firebase  integration but I really don't want that extra complexity as I already have a happily integrated nodejs backend. Just need the stripe flutter web front end. :(

**Reply by remonh87** [reply]
@tedfire  we have an example app that you can use,What do you miss more?  the web works the same as mobile but we support less payment methods at the moment. Furthermore you I recommend to read [stripe official docs](https://stripe.com/docs/payments/save-and-reuse?platform=react-native.)

**Reply by PatrikGolubar** [reply]
@tedfire 
As @remonh87 said. It works the same as mobile. For me it's working without problems and I have the same implementation as on mobile.

**Reply by nokia6290** [reply]
So does this mean stripe web is ready for production?

**Comment by PatrikGolubar** [comment]
How can I use future payments on web?

**Comment by tedfire** [comment]
Hi Remon,

Thanks for the reply.
Could you send me a link to the sample app please?

Many Thanks
Ted

**Reply by remonh87** [reply]
https://github.com/flutter-stripe/flutter_stripe/tree/main/example

**Comment by tedfire** [comment]
Hi Patrik,

Thanks for your response.

So using flutter_stripe_web I can't use presentPaymentSheet.
What should I use instead ?

Can you send me a link to an example app using flutter_strip_web please ?

Hope you can help.

Kind regards
Ted

**Reply by remonh87** [reply]
Paymentsheet does not work for web you need to use payment element. This is not something we chose but is how the stripe sdk works:  https://github.com/flutter-stripe/flutter_stripe/blob/main/example/lib/screens/payment_sheet/payment_element/payment_element.dart . The flow from user perspective is similar.
